cognite/client/data_classes/workflows.py

```diff
@@ -175,7 +176,7 @@ def _load(cls, resource: dict | str, cognite_client: CogniteClient | None = None
             external_id=function["externalId"],
             data=function.get("data"),
             # Allow default to come from the API.
-            is_async_complete=resource.get("isAsyncComplete"),  # type: ignore[arg-type]
+            is_async_complete=resource.get("isAsyncComplete", False),
```

Comment:
Load methods should not use defaults (if missing in the API response then it should be `None` here as well). I got schooled by Erlend on this a few PRs ago 😅 
```suggestion
            is_async_complete=resource.get("isAsyncComplete"),
```